packages/upswyng-web/src/App.tsx

```diff
@@ -37,6 +38,7 @@ class App extends Component {
             <Header />
             <Route exact path="/" component={Home} />
             <Route exact path="/about" component={About} />
+            <Route path="/coordinated-entry" component={CoordinatedEntry} />
             <Route path="/shelters" component={Shelters} />
             <Route path="/job_training" component={JobTraining} />
```

Comment:
we should update the `job_training` route to `job-training`

Reply:
true dat 👍  [donezo](https://github.com/CodeForBoulder/upswyng/pull/240/commits/53968e8944da7a0f61557ef47d4c43a5cb59c4bb)